run-all.sh

```diff
@@ -1,22 +1,9 @@
 #!/bin/bash
 
 # This script is a small convenience wrapper for running the doctrine testsuite against a large bunch of databases.
-# Just create the phpunit.xmls as described in the array below and configure the specific files <php /> section
-# to connect to that database. Just omit a file if you dont have that database and the tests will be skipped.
+# Create *.phpunit.xml files and specify database connection parameters in the <php /> section.
 
-configs[1]="mysql.phpunit.xml"
-configs[2]='postgres.phpunit.xml'
-configs[3]='sqlite.phpunit.xml'
-configs[4]='oracle.phpunit.xml'
-configs[5]='db2.phpunit.xml'
-configs[6]='pdo-ibm.phpunit.xml'
-configs[7]='sqlsrv.phpunit.xml'
-configs[8]='sqlanywhere.phpunit.xml'
-
-for i in "${configs[@]}"; do
-    if [ -f "$i" ];
-    then
-        echo "RUNNING TESTS WITH CONFIG $i"
-        phpunit -c "$i" "$@"
-    fi;
+for i in phpunit.xml.dist *.phpunit.xml; do
```

Comment:
Not really sure about the dist file here, probably shouldn't be included (also doesn't match description above).

Reply:
What's wrong with running the dist file first? Otherwise, in order to test `pdo-sqlite` one will need to create an extra file however the extension doesn't require configuration. The description doesn't say the script will run only those files.

Reply:
Besides it wasn't there previously, dist file is meant to be a template for script like this.